sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -385,6 +383,8 @@ def delete_configuration_setting(
                 key="MyKey", label="MyLabel"
             )
         """
+        etag = kwargs.pop("etag", "*")
```

Comment:
That's a change of behavior, before the default value was None. Expected?

Reply:
No. You are right. Bug.... Thx.